src/footer/mod.rs

```diff
@@ -0,0 +1,42 @@
+use leptos::*;
+
+use crate::icons::*;
+use crate::styles::*;
+
+enum Icon {
+    Docs,
+    Feedback,
+    Terms,
+    Support
+}
+
+#[component]
+pub fn Footer() -> impl IntoView {
+    let links = vec![
+        ("Docs", "https://docs.minaexplorer.com/", Icon::Docs),
+        ("Feedback", "https://discord.gg/Sc2JeqCPnX", Icon::Feedback),
+        ("Terms", "https://docs.minaexplorer.com/minaexplorer/website-terms-of-service", Icon::Terms),
```

Comment:
I wouldn't link to minaexplorer. `#` is better until we have the links of our own

Reply:
Nit: A more idiomatic way of representing the link:

```rust
pub struct Link(String, String, Icon);
// or
pub struct Link {
  label: String,
  href: String,
  icon: Icon,
}
```

Reply:
Okay I'll do that